cromwell.examples.conf

```diff
@@ -277,13 +277,13 @@ backend {
         # `temporary-directory` creates the temporary directory for commands.
         #
         # If this value is not set explicitly, the default value creates a unique temporary directory, equivalent to:
-        # temporary-directory = "mktemp -d \"$PWD\"/tmp.XXXXXX"
+        # temporary-directory = "$(mktemp -d \"$PWD\"/tmp.XXXXXX)"
```

Comment:
I think this example should be below the "what does this value mean" explanation below